include/config/pokemon.h

```diff
@@ -22,15 +22,16 @@
-#define P_EGG_MOVE_TRANSFER             GEN_LATEST  // Starting in Gen 8, if two Pokémon of the same species are together in the Daycare, one knows an Egg Move, and the other has an empty slot, the other Pokémon will receive the Egg Move in the empty slot. In Gen 9, if a Pokémon holds a Mirror Herb, it will receive Egg Moves from the other regardless of species.
+#define P_NIDORAN_M_DITTO_BREED        GEN_LATEST                   // Since Gen 5, when Nidoran♂ breeds with Ditto it can produce Nidoran♀ offspring. Before, it would only yield male offspring. This change also applies to Volbeat.
+#define P_INCENSE_BREEDING             GEN_LATEST                   // Since Gen 9, cross-generation Baby Pokémon don't require Incense being held by the parents to be obtained via breeding.
+#define P_EGG_HATCH_LEVEL              GEN_LATEST                   // Since Gen 4, Pokémon will hatch from eggs at level 1 instead of 5.
+#define P_BALL_INHERITING              GEN_LATEST                   // Since Gen 6, Eggs from the Daycare will inherit the Poké Ball from their mother. From Gen 7 onwards, the father can pass it down as well, as long as it's of the same species as the mother.
+#define P_TM_INHERITANCE               GEN_LATEST                   // Since Gen 6, the father no longer passes down TMs to the baby.
+#define P_MOTHER_EGG_MOVE_INHERITANCE  GEN_LATEST                   // Since Gen 6, the mother can also pass down Egg Moves.
+#define P_NATURE_INHERITANCE           GEN_LATEST                   // In Gen 3, Everstone grants Ditto and mothers a 50% chance to pass on Nature. Since Gen 4, anyone can pass on nature. Since Gen 5, the chance is 100%.
+#define P_ABILITY_INHERITANCE          GEN_LATEST                   // In B2W2, a female Pokémon has an 80% chance of passing down their ability if bred with a male. Since Gen 6, the chance is 80% for normal ability and 60% for Hidden Ability, and anyone can pass down their abilities if bred with Ditto. NOTE: BW's effect: 60% chance to pass down HA and random for normal ability has been omitted.
+#define P_EGG_MOVE_TRANSFER            GEN_LATEST                   // Starting in Gen 8, if two Pokémon of the same species are together in the Daycare, one knows an Egg Move, and the other has an empty slot, the other Pokémon will receive the Egg Move in the empty slot. In Gen 9, if a Pokémon holds a Mirror Herb, it will receive Egg Moves from the other regardless of species.
+#define P_SCATTERBUG_LINE_FORM_BREED   SPECIES_SCATTERBUG_ICY_SNOW  // Choose the Scatterbug form all Vivillon/Spewpa/Scatterbug will breed into, basically aligning with the "location" of the player's game.
```

Comment:
Is there a reason Icy Snow is the default rather than Fancy Pattern as is the vanilla precedent?
I would potentially add a static assert elsewhere to make sure that this define is a Scatterbug form and not something else, too.

Reply:
Icy snow is form 1 by index number, so we've always just had it as the default

Reply:
Also yeah I can add an assert

Reply:
That makes sense, but if we're adding a default form with SV as precedent, we should follow SV's form precedent as well imo.

Reply:
This config was gonna change the default used for the define as well initially, I just didn't really think about it and its kind of subjective. Might be easier to keep them the same by default

Reply:
Done